docs/resources/use_cases/vulcanexus_cloud/ddsrouter.yaml

```diff
@@ -0,0 +1,36 @@
+kind: Deployment
```

Comment:
NIT:
*In general for k8s yaml config files*
Maybe add some comments per important line? The raw configuration like this is very hard to read.
Maybe highlight some lines? Do not know if possible.
Check Example sections where a yaml is described to get a possible idea.

Reply:
I agree they are difficult to read, but I find it hard to add explanations in the deployment yamls without making it look like a Kubernetes tutorial. I think we can expect the readers of this example to be familiar with k8s.